docs/content/release-notes/flink-1.15.md

```diff
@@ -406,6 +406,13 @@ updating the client dependency to a version >= 7.14.0 is required due to interna
 The old JDBC connector (indicated by `connector.type=jdbc` in DDL) has been removed.
 If not done already, users need to upgrade to the newer stack (indicated by `connector=jdbc` in DDL).
 
+#### [FLINK-26126](https://issues.apache.org/jira/browse/FLINK-26126)
+
+New metric `numRecordsSend` has been introduced to denote the number of records sent to the external system 
+while the numRecordsOut will be used to denote the number of records transferred between tasks.
+
+Considering naming consistency, `numRecordsOutErrors` is deprecated, please use `numRecordsSendErrors` instead.
```

Comment:
WDYT about adding two different paragraphs? The first describes the perspective of a connector user that now needs to use `numRecordsSend` `numRecordsSendErrors` to monitor the records going to the external system.
The second paragraph holds information for connector developers that have to eventually update their connectors to use the new metrics because `numRecordsOurErrors` is deprecated and `numRecordsOut` has a different meaning.